product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx

```diff
@@ -1,56 +1,52 @@
 ---
-title: 'Identifying your image names'
+title: 'Identifying image names'
 ---
 
-For a quick installation with the aim of testing the product, see the [Quickstart](/postgres_distributed_for_kubernetes/latest/quickstart/). 
+For a quick installation with the aim of testing the product, see the [Quick start](/postgres_distributed_for_kubernetes/latest/quickstart/).
 For more targeted testing or production purposes, 
-the following section provides an overview of how to select a specific operand and proxy image version that goes in line with your Postgres distribution.
+this overview describes how to select a specific operand and proxy image version that's appropriate for your Postgres distribution.
 
 ## Operand image name 
 
 See [Operand images](/postgres_distributed_for_kubernetes/latest/private_registries/#operand-images) for general information about the images.  
 
 ### Name elements 
 
-The operand image name is composed of three elements: the postgres type, EDB Postgres Distributed, and the build version.
+The operand image name is composed of the Postgres type, EDB Postgres Distributed, their version numbers, and the build version.
 
-**The Postgres type:** The first step towards identifying your image names is knowing which Postgres type you are going to use, and how it translates to an image name. 
+**Postgres type:** The first part of the image name is the Postgres type you're using. The table shows how Postgres types correspond to this part of the image name.
 
 | Postgres type                | Image name            |
 |------------------------------|-----------------------|
 | EDB Postgres Advanced Server | edb-postgres-advanced |
 | EDB Postgres Extended Server | edb-postgres-extended |
 | PostgreSQL                   | postgresql            |
 
-**EDB Postgres Distributed:** The next step is identifying how Postgres Distributed translates to an image name. This is the same for all environment configurations: 
+**EDB Postgres Distributed:** For all environment configurations, `pgd` is the next part of the image name.
 
-| Product name             | Image name |
-|--------------------------|------------|
-| EDB Postgres Distributed | pgd        |
+**Version numbers:** The versions of the Postgres type and PGD, separated by a dash, are next in the image name.
 
-**Build version:** Last, the build version is 1, unless a new image is created in the scope of a security patch.
+**Build version:** The build version is 1, unless a new image is created in the scope of a security patch.
```

Comment:
What if it *is* created in the scope of the security patch?

Reply:
I wish I could be more clear here, but unfortunately there is no place the user can go to see a list of the existing builds. Given that this is the first version of the product, the team has not prioritized this at the moment, hence why I am being vague. 
Do you think this is a bit better? 

```suggestion
**Build version:** The build version is 1. If a new image is created to include a security patch, the build version is replaced by a new number in consecutive order, for example, 2.
```

Reply:
What if you just said:

**Build version:** Currently the only build version is 1.

Reply:
Looks good!